src/fatx.h

```diff
@@ -42,6 +43,11 @@
 #define FATX_STATUS_FILE_DELETED     1
 #define FATX_STATUS_END_OF_DIR       2
 
+#define FATX_RETAIL_CLUSTER_SIZE     (16 * 1024)
+#define FATX_RETAIL_PARTITION_COUNT  5
+
+#define FATX_READ_FROM_SUPERBLOCK    0
```

Comment:
I think it's not exactly clear that this should be specified in `fatx_open_device`'s `sectors_per_cluster` argument, and only in that argument. Can you add a comment here?